bundle-workflow/src/s3_utility.py

```diff
@@ -5,31 +5,35 @@
 # compatible open source license.
 
 import os
+from pathlib import Path
+from urllib.parse import urlparse
 
 import boto3
-
 from botocore.exceptions import ClientError
-from pathlib import Path
-from urllib.parse import urlparse
 
 
 def file_download_helper(bucket, key, path):
     try:
         bucket.download_file(key, path)
-    except ClientError as e: # TODO: handle the right exception
-        if e.response['Error']['Code'] == "404":
+    except ClientError as e:  # TODO: handle the right exception
+        if e.response["Error"]["Code"] == "404":
             print("The object does not exist.")
         else:
             raise
 
 
 class S3ReadWrite:
     def __init__(self, role_arn=None, role_session_name=None):
-        self.role_arn = role_arn if role_arn is not None else os.environ.get('JENKINS_S3_ROLE_ARN')
-        self.role_session_name = role_session_name \
-            if role_session_name is not None else os.environ.get('JENKINS_S3_ROLE_SESSION_NAME')
-        self.s3_resource = boto3.resource('s3')
-        self.s3_client = boto3.client('s3')
+        self.role_arn = (
+            role_arn if role_arn is not None else os.environ.get("JENKINS_S3_ROLE_ARN")
```

Comment:
The ENV var having "JENKINS" it it seems wrong.

Reply:
renamed it to `TEST_ORCHESTRATOR_ROLE_ARN`

Reply:
And why not `S3_ROLE_ARN` or `IAM_ROLE_ARN` or `ROLE_ARN`?

Reply:
Because it has specific permissions that are required by test-orchestrator workflow only. 

Read access for - build/ and bundles/
Write access for - test/

in the s3 bucket

Reply:
If the class is generic, environment variables should not be specific.

Reply:
I see your point now. Sure, will rename it to `ROLE_ARN`. Thx!